src/style.css

```diff
@@ -0,0 +1,85 @@
+@font-face {
+  font-family: "Avenir";
+  font-size: 16px;
+  font-weight: 300;
+  src: url(/fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-size: 16px;
+  font-weight: 900;
+  src: url(/fonts/Avenir-Heavy.ttf);
+}
+
+.search {
+  display: block;
+  width: 100%;
+  margin-top: 20px;
+}
+
+.input {
+  display: block;
+  box-sizing: border-box;
+  font-family: "Avenir", sans-serif;
```

Comment:
move `font-family: "Avenir", sans-serif;` to `html` tag and `font-family: inherit` here